docs/02-get-started/README.md

```diff
@@ -1 +1,27 @@
-This is the introductory document for this folder.
+---
+title: Get Started
+---
+
+This set of Get Started guides will show you how to set sail with Kyma and demonstrate its main use cases.
+
+All guides, whenever possible, demonstrate the steps in both kubectl and Kyma Dashboard.
+All the steps are performed in the `deafult` Namespace.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (1.19 or greater)
+- [curl](https://github.com/curl/curl)
+- [k3d](https://k3d.io)
+- [Kyma CLI](../04-operation-guides/operations/01-install-kyma-CLI.md)
+
```

Comment:
I think we also need Docker installed right?

Reply:
`k3d` is Kubernetes in Docker. To install k3d you need Docker, but it's a prerequisite for k3d, not for Kyma per se, so it's not listed here.